### PR TITLE
Implement the event website block frontend view

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/109051

Implement the event website block frontend view

### CODE REVIEW

File: src/modules/views/event-website/style.pcss [hunk 1]
```diff
@@ -3,3 +3,24 @@
  *
  * @since  TBD
  */
+
+.tribe-block__event-website {
+	padding: 4% 20px;
+	align-items: center;
+	border: 1px solid #E1E3E6;
+	display: inline-flex;
+	font-family: 'Helvetica', 'sans-serif';
```
Comment (bordoni) [inline src/modules/views/event-website/style.pcss]: We should avoid font-family for front-end stuff.

File: src/modules/views/event-website/style.pcss [hunk 2]
```diff
@@ -3,3 +3,24 @@
...
+	justify-content: center;
+	min-height: 54px;
+	padding: 0 20px;
+	font-size: 1.125rem;
+	background-color: #009FD4;
+	max-width: 100%;
+	color: #fff;
+
+	&,
```
Comment (bordoni) [inline src/modules/views/event-website/style.pcss]: Typo?

**Comment by bordoni** [comment]
Also changelog

File: src/modules/views/event-website/style.pcss (hunk 1 above)
Comment (juanfra) [reply src/modules/views/event-website/style.pcss]: 👍

File: src/modules/views/event-website/style.pcss (hunk 2 above)
Comment (juanfra) [reply src/modules/views/event-website/style.pcss]: nope, actually a code that I left before adding the color to the class. `&` means self, so the idea was to assign the color property to both the class and the hover state. Removing that.

**Comment by juanfra** [comment]
> Should we split that Border, padding and Background color into something re-usable?

Like a "size" class, and border class or something like that?
i.e
`.our-button.medium {
   padding: X Y;
}
.our-button.bordered {
   border: 1px solid black;
}
`

**Comment by bordoni** [comment]
Possibly into a global rule for that style of block. Not sure tho.

File: src/views/blocks/event-website.php [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
-    class="tribe-block__website"
-    href="<?php echo esc_url( $attributes['href'] ); ?>"
-    target="<?php echo esc_attr( $target ); ?>"
-    <?php if ( '_blank' === $target  ) : ?> rel="noopener noreferrer" <?php endif; ?>
-    >
-    <?php echo esc_html( $attributes['urlLabel'] ); ?>
-</a>
+<div class="tribe-block tribe-block__event-website">
+    <a
+        class="blue bordered"
```
Comment (bordoni) [inline src/views/blocks/event-website.php]: We need to avoid these very generic classes since this is on the front-end.

Comment (juanfra) [reply src/views/blocks/event-website.php]: I thought that's what you asked.

**Pull request merged**